lib/analyzer/binary-version-extractors/openjdk-jre.ts

```diff
@@ -30,11 +30,14 @@ function parseOpenJDKBinary(fullVersionOutput: string) {
   */
   const jdkVersionLines = fullVersionOutput &&
                           fullVersionOutput.trim().split('\n');
-  if (!jdkVersionLines || jdkVersionLines.length !== 3) {
```

Comment:
@leongold not very critical, but removing this condition doesn't have anything to do with adopt openjdk, right? we purposely added this to be restrictive after checking many cases of `java -version` outputs on different images. @darmalovan

Reply:
This PR fixes adopt openjdk (I hope :D) but it's more of a happy side-effect in this context.

The main purpose is to alleviate the maybe too restrictive parsing and, in the hopes that other flavours of openjdk will be supported out of the box.

We have just 1 use case of it now, but perhaps when we encounter more cases we'll find out this solution isn't quite right either...

I think that all in all parsing strings can't actually be a fail proof method, but as long as this is what we do, we can only tweak restrictions when we encounter new use cases...

Reply:
I guess it's a point of view if adding support for adopt openjdk is a fix or not.
I agree regards parsing strings but we need to make sure our rules are not too loose...

Reply:
either way just `improve` isn't informative :)

Reply:
I agree, I'll improve `improve` 😄